gammapy/datasets/make.py

```diff
@@ -64,3 +72,158 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory_name, n_obs, debug=False):
+    """Make a test observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory_name : string
+    	name of the observatory; a list of choices is given in `~gammapy.obs.observatory_locations`
+    n_obs : integer
+    	number of observations for the obs table
+    debug : bool
+    	show UTC times instead of seconds after the reference
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # build a time reference as the start of 2010
+    dateref = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    # TODO: using format "iso" for `~astropy.Time` for now; eventually change it
+    # to "fits" after the next astropy stable release (>1.0) is out.
+    dateref_mjd_fra, dateref_mjd_int = np.modf(dateref.mjd)
+
+    # header
+    header = {'OBSERVATORY_NAME': observatory_name,
+              'MJDREFI': dateref_mjd_int, 'MJDREFF': dateref_mjd_fra}
+    ObservationTable.meta = header
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    obs_table.add_column(col_obs_id)
+
+    # TODO: `~astropy.table.Table` doesn't handle `~astropy.time.TimeDelta` columns
+    # properly see:
+    #  https://github.com/astropy/astropy/issues/3832
+    # until this issue is solved (and included into a stable release), quantity
+    # objects are used.
+
+    # obs time: 30 min
+    time_observation = TimeDelta(30.*60.*np.ones_like(col_obs_id.data), format='sec')
+    time_observation = Quantity(time_observation.sec, 'second') # converting to quantity
+    col_time_observation = Column(name='TIME_OBSERVATION', data=time_observation)
+    obs_table.add_column(col_time_observation)
+
+    # livetime: 25 min
+    time_live = TimeDelta(25.*60.*np.ones_like(col_obs_id.data), format='sec')
+    time_live = Quantity(time_live.sec, 'second') # converting to quantity
+    col_time_live = Column(name='TIME_LIVE', data=time_live)
+    obs_table.add_column(col_time_live)
+
+    # start time
+    # random points between the start of 2010 and the end of 2014
+    # using the start of 2010 as a reference time for the header of the table
+    # observations restrict to night time
+    # considering start of astronomical day at midday: implicit in setting the
+    # start of the night, when generating random night hours
+    datestart = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    dateend = Time('2015-01-01 00:00:00', format='iso', scale='utc')
+    time_start = Time((dateend.mjd - datestart.mjd)*np.random.random(len(col_obs_id)) + datestart.mjd, format='mjd', scale='utc')
```

Comment:
Please roughly respect the 80 character line length limit.
Use temp variables to break up complex expressions in pieces.

Running pep8 on the files you've edited will show you this as a PEP8 violation.
Good Python editors / IDEs like PyCharm will also directly indicate this to you in the editor.